vnext/ReactUWP/Utils/Helpers.cpp

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "pch.h"
+#include "Helpers.h"
```

Comment:
>Helpers [](start = 10, length = 7)

Do you intent to put all helpers in this file?  Our other things are named  SomethingUtils.h with some indication of what kind of utils you're getting.  Helpers.h is super generic

Reply:
Yes. for some small functions, we don't need to spread across multiple files. and I suggest we just dump them to this file.

Reply:
I don't feel too strongly either way, but I would like to be _consistent_ in our conventions.  If your intent is to remove XXXUtils then please log a bug to do the consolitation so we don't lose track.

---
In reply to: [295434692](https://github.com/microsoft/react-native-windows/pull/2623#discussion_r295434692) [](ancestors = 295434692)